### PR TITLE
Additional Inovelli device support 

### PR DESCRIPTION
**Pull request by InovelliUSA**
LZW31 - Basic Dimmer
LZW31-SN - Red Series Dimmer
LZW40 - Dimmable Bulb
LZW41 - Tunable White Bulb
LZW41 - RGBW Bulb

### CODE REVIEW

File: config/inovelli/lzw31-sn.xml [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+  <CommandClass id="133">
+    <Associations num_groups="4">
+      <Group index="1" label="Lifeline" max_associations="5" />
+      <Group index="2" label="Basic_Set" max_associations="5" />
+	  <Group index="3" label="Switch_MultiLevel_Set" max_associations="5" />
+      <Group index="4" label="Switch_MultiLevel_Start/Stop" max_associations="5" />
+    </Associations>
+  </CommandClass>
+  <!-- Central Scene Reports -->
+  <CommandClass id="91">
```
Comment (Fishwaldo) [inline config/inovelli/lzw31-sn.xml]: This should be auto discovered in OZW 1.6. Is something not working here?

File: ] [hunk 2]
```diff
@@ -0,0 +1,207 @@
+<Product Revision="1" xmlns="https://github.com/OpenZWave/open-zwave">
```
Comment (rjbez17) [inline ]]: This file looks like it was accidentally pushed?

**Comment by Fishwaldo** [comment]
ping?

**Comment by zacs** [comment]
@InovelliUSA ping? Hoping to add a ton of RGB bulbs to my zwave2mqtt instance...

**Comment by mdallaire** [comment]
I used the following process to confirm that it works without the commandclass <CommandClass id="91">
With zwave2mqtt v.3.0.1:

1. Copy over the Inovelli configuration
2. Remove the <CommandClass id="91"> section from lzw31-sn.xml
3. Let the devices be discovered and refreshed
4. Trigger central scene
`z2m:Zwave zwave node 55: changed: 91-1-3:Scene 3:Inactive -> Pressed 1 Time +568ms
z2m:Zwave zwave node 55: changed: 91-1-3:Scene 3:Pressed 1 Time -> Inactive +1s`
5. Check resulting config xml for <CommandClass id="91" name="COMMAND_CLASS_CENTRAL_SCENE"> (it is present):
https://gist.github.com/mdallaire/895cc99a20fe14ef7b7589f8d27fb882

Let me know if you need additional details.

**Comment by Fishwaldo** [comment]
Thanks!

**Pull request merged**